compiler/quilt/test/test_command.py

```diff
@@ -523,41 +523,104 @@ def test_rm_doesnt_break_cache(self):
         assert isinstance(bar.foo(), pd.DataFrame)
 
     def test_export(self):
+        # pathlib will translate paths to windows or posix paths when needed
         Path = pathlib.Path
         pkg_name = 'testing/foo'
-        pkg_name_subpkg = pkg_name + '/subdir'
+        subpkg_name = 'subdir'
+        single_name = 'single_file'
+        single_bytes = os.urandom(200)
 
-        # pathlib will translate these to windows paths
-        test_data = [
-            ('example', os.urandom(500)),
-            ('subdir/subdir_example', os.urandom(300)),
-            ('readme', os.urandom(200)),
+        subdir_test_data = [
+            (subpkg_name + '/subdir_example', os.urandom(300)),
+            (subpkg_name + '/9bad-identifier.html', os.urandom(100)),
             ]
 
+        test_data = [
+            (single_name, single_bytes),
+            ('readme.md', os.urandom(200)),
+            # these are invalid python identifiers, but should be handled without issue
+            ('3-bad-identifier/bad_parent_identifier.html', os.urandom(100)),
+            ('3-bad-identifier/9{}bad-identifier.html', os.urandom(100)),
+            ] + subdir_test_data
+
```

Comment:
also add test cases for:
- overwriting existing dirs/files
- improper permissions to create the subdir
- (if easy) full disk